.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 99
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: weekly
```

If you want it less frequently
```suggestion
      interval: monthly
```

---

Yes, let's go for monthly 👍